nuxeo-retention-web/elements/nuxeo-retention-analytics.js

```diff
@@ -0,0 +1,351 @@
+            </div>
+            <nuxeo-date-picker value="{{startDate}}"></nuxeo-date-picker>
+            <div class="dateLabel">
+              <label>[[i18n('retentionAnalytics.to')]]</label>
+            </div>
+            <nuxeo-date-picker value="{{endDate}}"></nuxeo-date-picker>
+          </div>
+        </nuxeo-card>
+
+        <div class="flex-layout">
+          <!-- Retention by Document Type -->
+          <nuxeo-repository-data
+            start-date="[[_formatDate(startDate)]]"
+            end-date="[[_extendEndDate(endDate)]]"
+            date-field="dc:created"
+            grouped-by="ecm:primaryType"
+            where='{"ecm:isRecord": "true"}'
+            group-limit="20"
+            data="{{allDocCategoriesCount}}"
+            index="[[index]]"
+          >
+          </nuxeo-repository-data>
+
+          <nuxeo-card heading="[[i18n('retentionAnalytics.retentionByDocType')]]">
+            <chart-pie
+              values="[[_values(allDocCategoriesCount)]]"
+              labels="[[_labels(allDocCategoriesCount)]]"
+              options='{ "legend": { "display": true, "position": "bottom", "labels": { "boxWidth": 12 } }, "animation": false }'
+            >
+            </chart-pie>
+          </nuxeo-card>
+
+          <!-- RETENTION STATUS -->
+          <!-- No Retention -->
+          <nuxeo-repository-data
+            start-date="[[_formatDate(startDate)]]"
+            end-date="[[_extendEndDate(endDate)]]"
+            date-field="dc:created"
+            ecm-mixin-type="HasRelatedText"
+            metrics="cardinality(ecm:uuid)"
+            where='{"ecm:isRecord": "false"}'
+            data="{{noRetention}}"
+            index="[[index]]"
+          >
+          </nuxeo-repository-data>
+
+          <!-- Future Retentions -->
+          <nuxeo-repository-data
+            start-date="[[_formatDate(startDate)]]"
+            end-date="[[_extendEndDate(endDate)]]"
+            date-field="dc:created"
+            grouped-by="ecm:retainUntil"
+            group-limit="10"
+            data="{{futureRetention}}"
+            index="[[index]]"
+          >
+          </nuxeo-repository-data>
+
+          <!-- Expired Retentions -->
+          <nuxeo-repository-data
+            start-date="[[_formatDate(startDate)]]"
+            end-date="[[_extendEndDate(endDate)]]"
+            date-field="dc:created"
+            grouped-by="record:retainUntil"
+            group-limit="10"
+            data="{{expiredRetention}}"
+            index="[[index]]"
+          >
+          </nuxeo-repository-data>
+
+          <nuxeo-card heading="[[i18n('retentionAnalytics.retentionStatus')]]">
+            <chart-pie
+              values="[[_valuesRetentionStatus(noRetention, futureRetention, expiredRetention)]]"
+              labels="[[_labelsRetentionStatus()]]"
+              options='{ "legend": { "display": true, "position": "bottom", "labels": { "boxWidth": 12 } }, "animation": false }'
```

Comment:
I think, they are the same options ` options='{ "legend": { "display": true, "position": "bottom", "labels": { "boxWidth": 12 } }, "animation": false }'` on each chart, perhaps extract it as a const.

Reply:
by extracting it as a const, does it mean it will be a read-only property? Or is there another way to extract it?